Session/Meta/AppDelegate.swift

```diff
@@ -40,4 +40,20 @@ extension AppDelegate {
     @objc func stopClosedGroupPoller() {
         ClosedGroupPoller.shared.stop()
     }
+    
+    @objc func getAppModeOrSystemDefault() -> AppMode {
+        let userDefaults = UserDefaults.standard
+        
+        guard userDefaults.dictionaryRepresentation().keys.contains("appMode") else {
```

Comment:
Normally I'd use an `if` statement here rather than a `guard`, because `guard` (at least how we tend to use it) is meant more for precondition checks rather than direct control flow.

Reply:
Gotcha - that makes sense. Shall keep that in mind for next time